script/chart_sync.sh

```diff
@@ -15,20 +15,19 @@
 
 set -e
 
-CHARTS_REPO="bitnami/charts"
 source $(dirname $0)/chart_sync_utils.sh
 
 user=${1:?}
 email=${2:?}
 if changedVersion; then
```

Comment:
As mentioned elsewhere, I'm not sure we can rely on `changedVersion`, though maybe we can use semver to have something similar while still allowing local version changes in devel.

Reply:
+1 to the semver detection to avoid race conditions. But, that said, I still think is approach is valid as long we accept and merge PR sequentially.